packages/neo-one-client-switch/src/browser/LedgerTransport.ts

```diff
@@ -0,0 +1,12 @@
+import TransportU2F from '@ledgerhq/hw-transport-u2f';
+import { Ledger } from '../common';
+
+const ledgerU2FTransport: Ledger = {
```

Comment:
I would just do `export const LedgerTransport: Ledger = {}` rather than renaming it below.